frontend/src/features/permits/components/form/ContactDetails.tsx

```diff
@@ -91,8 +94,7 @@ export const ContactDetails = ({ feature }: { feature: string }) => {
         </div>
 
         <InfoBcGovBanner
-          description="The permit will be sent to the email listed below in addition to the email on your 
-      onRouteBC Company Information. If you would also like to receive the permit by Fax, please enter a fax number."
+          msg="The permit will be sent to the email on your onRouteBC Company Information, in addition to any email listed below. If you would also like to receive the permit by Fax, please enter a fax number."
```

Comment:
maybe a `const` here? I know we're supposed to keep the mention of the fax in there, but I don't like it if we don't support it :(